johnsonandjohnson/Bodiless-JS
# feat:(test-site, components): Menu items need to support simple editor

## Changes
- Navigation Menu items can now be superscripted.
- Design Tokens for Top Menu and Burger Menu are now consolidated into a single `token.ts` file.
- Navigation Menu now has all site links
- Responsive Breakpoints are now coming from the site `tailwind.config.js` and not hardcoded.

## Test Instructions
- Make sure Menu Items and Sub Items can be superscripted.
- Make sure the menu on the `/rcmenu` page is full width.
- The rest of the functionality is unchanged ( same as master )

## Related Issues
Closes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/287
